### PR TITLE
Update ember-cli to 1.13.8, use simplified initializer pattern

### PR DESCRIPTION
**Pull request by mike-north**
In canary, the initializer was was throwing a deprecation warning due to: https://github.com/emberjs/ember.js/pull/11440

### CODE REVIEW

File: app/initializers/export-application-global.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import Ember from 'ember';
 import config from '../config/environment';
 
-export function initialize(container, application) {
+export function initialize() {
+  var application = arguments[1] || arguments[0];
```
Comment (mike-north) [inline app/initializers/export-application-global.js]: @rwjblue @stefanpenner if this is an acceptable fix, I'm happy to apply this pattern to ember's internal initializers. Canary is currently throwing warnings, so I assume they'll need to be handled before cutting the next beta

Comment (rwjblue) [inline app/initializers/export-application-global.js]: @mike-north - Are you saying that the initializers that are internal to Ember itself are triggering this deprecation?  Can you open an issue for that (the fix in Ember itself isn't quite like this though)?

Comment (mike-north) [inline app/initializers/export-application-global.js]: > Are you saying that the initializers that are internal to Ember itself are triggering this deprecation?

Yes

> Can you open an issue for that

I'll open a PR. The particulars of the fix are likely to be simple enough that I don't mind having to re-do them after feedback

Comment (rwjblue) [inline app/initializers/export-application-global.js]: Sounds good. The solution in Ember is to import `isEnabled` and use different signatures based on the flag being enabled. Also, I'm concerned that our tests did not show this (we throw errors if we hit a deprecation).

**Comment by acorncom** [comment]
Just ran into this bug when working on running the LiquidFire test server (as I'm starting on some docs changes here: https://github.com/emberjs/guides/issues/448#issuecomment-131980600), which also works against `canary` ...

**Comment by rwjblue** [comment]
Released as 1.0.4.

**Comment by mike-north** [comment]
:+1:

**Pull request merged**